examples/production/README.md

```diff
@@ -1,19 +1,20 @@
 # Recommended production setup
 
-To install Spire with the least privileges possible we deploy spire across 2 namespaces.
+## CRD Deployment
```

Comment:
Trying to get rid of this entire file here: https://github.com/spiffe/helm-charts-hardened/pull/237